Rentals (#41)
decentraland/documentation: content/player/marketplace/rentals.md

![](/images/Rentals/Image3.png)

You can also set an expiration date for the listing. This means that, if the LAND wasn’t rented until the selected date, the listing will be removed from the Marketplace. Also, the smart contract will reject the expired signature so that no one can rent it for the listing price and duration previously selected. This is a security measure to prevent it to be rented for an undesired price or duration. 

![](/images/Rentals/Image4.png)

After the price, rent period and listing expiration date are set, your LAND will appear as available for rent in the Marketplace.

<aside>
💡 When LAND is rented by a Tenant, it can not be sold until it’s claimed back. Bids from potential buyers can not be received either.

</aside>

<aside>
💡 Voting Power is kept by the LAND Owner, even if it is rented.

</aside>

## Edit or Cancel a Listing

After the LAND is Listed for Rent in the Marketplace, and before anybody rents it, you can edit the conditions of the Listing by clicking on the pencil icon in the LAND detail. You can also remove the Listing from the Marketplace and the blockchain.

![](/images/Rentals/Image5.png)

<aside>

> 💡 Edit and cancel actions may require a transaction, which costs gas. See Transactions section below for more details.

Reviewer: ```suggestion
💡 Edit and cancel require a transaction, which costs gas. See Transactions section below for more details.
```

</aside>

## After the Rent is over

After the Rent is over, you can either **Claim your LAND Back, or List it for Rent Again**.

**Operator Permissions are not transferred automatically back to the LAND Owner**. In order to get them back, the LAND Owner has to Claim the LAND back by sending that transaction and paying for the gas fee. Confirming the transaction will take out Operator Permissions from the Tenant and give them back to the LAND Owner.

![](/images/Rentals/Image6.png)

The other possibility is to List the LAND for Rent Again, instead of claiming it back. This will not require paying for another transaction, but **Operator Permissions will be kept by the previous Tenant until a new Tenant confirms a new Rent.** 

The LAND Owner can edit the price, rent period, and listing expiration date for the new listing. 

![](/images/Rentals/Image7.png)

Both actions can be done from the LAND detail page in the Marketpalce.

![](/images/Rentals/Image8.png)

## Renting Status

> You can check the Status of any rented LAND in My Store > My Assets. The possible status are:

Reviewer: ```suggestion
You can check the Status of any rented LAND in My Assets > Store > On Rent. The possible status are:
```

But, not every action involved requires an entry in the blockchain. If that was the case, it would be too expensive for both parties. 

Transactions in the blockchain are minimum in order to provide a **robust and trustless system for LAND renting while keeping it affordable.** These are all the transactions to consider:

## For Land Owners

### List for Rent

Before Listing the first Parcel or Estate for rent, LAND Owners need to allow the Rents Smart Contract to operate LAND on their behalf. This has to be done only once for Parcels and only once for Estates. 

![](/images/Rentals/Image16.png)

### Claim LAND Back or List for Rent Again

After the renting period ends, **Operator Permissions are not transferred automatically back to the LAND Owner**. In order to get them back, the LAND Owner has to Claim the LAND back by sending that transaction and paying for the gas fee.

![](/images/Rentals/Image17.png)

Another possibility is to List the LAND for Rent Again, instead of claiming it back. This will not require paying for another transaction, but Operator Permissions will be kept by the previous Tenant until a new Tenant confirms a new Rent. 

### Edit Listing

> If either the Price, Rent Period, or Expiration Date is changed, a transaction has to be sent by the LAND Owner in order to protect themselves from somebody hitting the Smart Contract directly (not from the Marketplace UI) and getting it from a lower price than desired or for an undesired duration. 

Reviewer: ```suggestion
If either the Price, Rent Period, or Expiration Date is changed, a transaction has to be sent by the LAND Owner in order to protect themselves from somebody using the previous listing signature on the Smart Contract directly (not from the Marketplace UI) and getting it from a lower price than desired or for an undesired duration. 
```
